src/Product/AttributeMapping/Traits/IsFieldTrait.php

```diff
@@ -94,12 +94,12 @@ public static function get_source_taxonomies(): array {
 	public static function get_source_product_fields(): array {
 		$fields = [
 			'product:backorders'       => __( 'Allow backorders setting', 'google-listings-and-ads' ),
-			'product:product_title'    => __( 'Product title', 'google-listings-and-ads' ),
+			'product:title'            => __( 'Product title', 'google-listings-and-ads' ),
 			'product:sku'              => __( 'SKU', 'google-listings-and-ads' ),
-			'product:stock_qty'        => __( 'Stock Qty', 'google-listings-and-ads' ),
+			'product:stock_quantity'   => __( 'Stock Qty', 'google-listings-and-ads' ),
 			'product:stock_status'     => __( 'Stock Status', 'google-listings-and-ads' ),
 			'product:tax_class'        => __( 'Tax class', 'google-listings-and-ads' ),
-			'product:variation_title'  => __( 'Variation title', 'google-listings-and-ads' ),
+			'product:name'             => __( 'Variation title', 'google-listings-and-ads' ),
```

Comment:
As far as I tested it, when I use `name` it gets the variation title. Whereas `title`, gets always the parent title. So we are not removing variation, it's just that with `name` key, since to be enough to select the variation...

Reply:
Thanks for clarifying, I see we are actually using those to retrieve the product props and `variation_title` isn't a function. Although if you choose that mapping it can still be confusing to users what's actually going to be displayed, since it depends on the product type and the length/count of attributes.

```
Simple product:
title = "Simple Product Name", name = "Simple Product Name"

Variation product (1 attribute):
title = "Parent Product Name", name = "Parent Product Name - Attribute Name"

Variation product (more than 2 attributes or long attribute names):
title = "Parent Product Name", name = "Parent Product Name"
```

But I'm fine with leaving it in, although it will probably rarely be used.